docs/README.md

```diff
@@ -29,4 +29,9 @@ Please see detailed instructions [here](assessment_instructions.md).
 
 ## Acknowledgments
 
-This software was developed in collaboration with the United Nations World Food Programme (WFP) Innovation Accelerator. The WFP Innovation Accelerator identifies, supports and scales high potential solutions to hunger worldwide. The Innovation Accelerator supports WFP innovators and external start-ups and companies through financial support, access to a network of experts and a global field reach.
+This software was developed in collaboration with the United Nations World Food Programme (WFP) Innovation Accelerator and [InstaDeep](https://www.instadeep.com/)'s AI for Social Good (AI4SG) team.
+
+The WFP Innovation Accelerator identifies, supports and scales high potential solutions to hunger worldwide. The Innovation Accelerator supports WFP innovators and external start-ups and companies through financial support, access to a network of experts and a global field reach.
+InstaDeep's AI for Social Good (AI4SG) team utilizes the company's expertise in AI research and engineering to create technologies aimed at improving human welfare and enhancing global well-being.
+
+![collaborators logos](https://drive.google.com/file/d/1UaizEHM4V-PSSKphoIFvTnSa7e1dOt1Z/view?usp=sharing)
```

Comment:
Can you add this image file to the repo? Referencing a Google drive link seems a bit unstable.

Reply:
Hi @jzxu 
I added that link as a placeholder. I see there's a storage (https://storage.googleapis.com/skai-public/documentation_images) for other images used in the docs. Wondering if you could add this as well as I don't have access.

Reply:
Thank you @jzxu 
I have added the image to the repo as suggested.